client/tests/webdriver/specs/editLocation.spec.js

```diff
@@ -0,0 +1,48 @@
+import EditLocation from "../pages/location/editLocation.page"
+
+const EX_LOCATION_ID = "f2207d9b-204b-4cb5-874d-3fe6bc6f8acd"
```

Comment:
As you are not testing anything specific to the edit page, you could also just use the create page.
That way you get rid of a hard-coded location id and of the problem of setValue appending a value instead of replacing an existing one.

Reply:
I added a creation test but i should probably keep a editLocation test as well instead of removing completely. So how should i approach if i don't want to use hard-coded Location uuid? Create a new one on the editLocation test maybe?

Reply:
The location is fairly simple to create, so i will create first and then edit.